web/js/utils/auth.js

```diff
@@ -1,21 +1,7 @@
-/*
```

Comment:
We do need to keep these headers?  Google requires them on code hosted under its GitHub account.  Make sure they exist on any new source files that you add, as well.